<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibManagement</class>
 <widget class="QMainWindow" name="LibManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#F3F6FD;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>95</y>
      <width>100</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/navigator.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>250</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>造字工房情书体（非商用）</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LibManagement</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>850</width>
      <height>490</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/frame.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Bio">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>530</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>思源黑体</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/topBar.png);</string>
    </property>
    <property name="text">
     <string>    Bio显示区域</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_home">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>120</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;首页&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:black; 
	background-color:#FFF4EA;
    border: 3px;
    border-radius: 30px;
}

QPushButton:hover{
	background-color:#FF9465;
	color:rgb(0,0,0);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;搜书&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:black; 
	background-color:#E9FCF1;
    border: 3px;
    border-radius: 30px;	

}

QPushButton:hover{
	background-color:#34C471;
	color:rgb(0,0,0);
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_me">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;我的信息&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:black; 
	background-color:#EFEFEF;

    border: 3px;
    border-radius: 30px;
}

QPushButton:hover{
	background-color:#FFD400;
	color:rgb(0,0,0);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/hr.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/hr.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_logout">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;注销登录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:black; 
	background-color:#F6F5FE;

    border: 3px;
    border-radius: 30px;
}

QPushButton:hover{
	background-color:#4F3FF0;
	color:rgb(0,0,0);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;退出软件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color:black; 
	background-color:#FFEDF3;
    border: 3px;
    border-radius: 30px;
}

QPushButton:hover{
	background-color:rgb(255, 166, 124);
	color:rgb(0,0,0);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>3</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   color:black; 
background-color:white;
	border-radius: 12px;
}
QPushButton:hover{
  background-color:rgb(85, 170, 255);
  color:rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>➖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>3</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
   color:black;
border-radius: 12px;
background-color:white;
}
QPushButton:hover{
  background-color:rgb(212, 64, 39);
  color:rgb(0,0,0);
}</string>
    </property>
    <property name="text">
     <string>❌</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_avatar">
    <property name="geometry">
     <rect>
      <x>917</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;头像&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
